src/main/java/hudson/tasks/junit/JUnitResultArchiver.java

```diff
@@ -75,6 +75,12 @@
     private boolean keepLongStdio;
 
     /**
+     * If true, and if no JUnit output, don't fail the JUnit reporting.
+     * @since 1.596.2	 
```

Comment:
The current plugin version is 1.10-SNAPSHOT . Presumably it will be merged into 1.10, so makes sense to add this version. BTW the recommended practice is just to use <code>@since TODO</code> and leave to the person who integrates the PR